pipelines/skylab/optimus/scripts/parse_terra.py

```diff
@@ -0,0 +1,121 @@
+import argparse
```

Comment:
@kishorikonwar Is this the same script (more or less) as the "parse_terra_tsv.py" script that is already checked in? Let's not have the same script checked in twice please. Also I think the other one is more up-to-date, can we please make sure the workflow-controller calls the right one?